front-end/src/domains/GpuServerDetailSummary/GpuServerDetailSummary.tsx

```diff
@@ -28,42 +32,37 @@ const GpuServerDetailSummary = ({
         aria-label={`서버 상태: ${detail.isOn ? "ON" : "OFF"}`}
       />
     </ServerNameBox>
-    <div>
-      <Text as="h4" weight="bold">
+    <StyledGpuServerDetailSummaryList>
```

Comment:
```suggestion
    <DetailSummaryList>
```

또는 

```suggestion
    <SummaryList>
```

Styled + 컴포넌트명의 컨벤션은, 
- 최상단의 컴포넌트는 Styled + 컴포넌트명
- 기존에 존재하는 컴포넌트에 스타일을 덧씌울 때
였던거 같아요. 
 
나머지 그냥 일반적인 스타일드 컴포넌트에는 굳이 Styled를 안붙여도 되지 않을까 싶은데, 어떻게 생각하시나요~?

Reply:
> 기존에 존재하는 컴포넌트에 스타일을 덧씌울 때였던거 같아요.

요런 의미가 내포되어 있는 줄 몰랐네요 ! 동의합니다 👍

Reply:
59e4be1